.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Maybe best to avoid adding editor-specific files to the repository in general? Is there a reason to include this one?